packages/playground/src/components/openapi-output.tsx

```diff
@@ -0,0 +1,28 @@
+import { FunctionComponent, useCallback, useState } from "react";
+import { OutputEditor } from "./cadl-editor";
+import { PlaygroundManifest } from "../manifest";
+import { SwaggerUI } from "./swagger-ui";
+export interface OpenAPIOutputProps {
+  content: string;
+}
+
+export const OpenAPIOutput: FunctionComponent<OpenAPIOutputProps> = (props)=> {
+  const[selected, setSelected] = useState<"raw"|"swagger-ui">("raw");
+  const handleSelected = useCallback(
+    () => { setSelected(selected === "raw"?("swagger-ui"):("raw"));
+    },
+    [selected],
+  );
+  return(
+    <>
+    {PlaygroundManifest.enableSwaggerUI? (
+    <select className="output-dropdown" onChange={handleSelected} value={selected}>
+      <option>OpenApi</option>
```

Comment:
Should also make dropdown floating above on the right 
![image](https://user-images.githubusercontent.com/1031227/180528505-ba5d3442-92d3-4bbb-8429-6b111f702b36.png)